recipes/apache-flink/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "apache-flink" %}
+{% set version = "1.18.0" %}
+
+package:
+  name: "apache-flink-python"
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/apache-flink-{{ version }}.tar.gz
```

Comment:
@gforsyth Is this URL correct? I get a 0 byte file from https://pypi.io/packages/source/a/apache-flink-python/apache-flink-1.18.0.tar.gz

Think it should resolve to https://pypi.io/packages/source/a/apache-flink/apache-flink-1.18.0.tar.gz, but maybe you're just waiting on the name decision.

Reply:
The URL is correct, but yeah, it's confusing.  The `name` variable is set on the first line of the recipe to `apache-flink` (and it will stay that way so this resolves correctly).  The value of the `name` field under `package` is what will be used as the name for installation, i.e. what you pass to `conda install ...`

Reply:
OK, ignore my suggestion below. :)